cs3api4lab/api/share_api_facade.py

```diff
@@ -128,23 +132,30 @@ def list_grantees_for_file(self, file_path):
         :param file_path: path to the file
         :return: list of grantees
         """
-        share_list = self.share_api.list()
+
+        file_path = FileUtils.remove_drives_names(file_path)
+        file_path = FileUtils.check_and_transform_file_path(file_path)
+        file_stat = self.file_api.stat_info(file_path)
+        share_filter = {
```

Comment:
This is what I was talking with you, I don't like the abstraction this high in the stack. These CS3APIs details should be hidden as much as possible lower, like in the cs3 client, and then we can expose something a bit more high level.
Why don't we just call a  new method, i.e `self.share_api.file_shares`, and pass the file path (or the stat info), and we build the filter inside?